test/functionalTest/cases/0000_ngsi10_update/update_sequence_empty_value_in_context_value.test

```diff
@@ -72,8 +72,8 @@ Date: REGEX(.*)
         </contextAttributeList>
       </contextElement>
       <statusCode>
-        <code>472</code>
-        <reasonPhrase>request parameter is invalid/not allowed</reasonPhrase>
+        <code>422</code>
+        <reasonPhrase>Invalid Modification</reasonPhrase>
         <details>action: APPEND - entity: [OfficeRoom, Room, false] - offending attribute: pressure - empty attribute not allowed in APPEND or UPDATE</details>
       </statusCode>
     </contextElementResponse>
```

Comment:
Could you include a test harness to test the fixing in the NGSIv2 API, pls? Contact me if you have doubt about it :)